change/@microsoft-teams-js-adfbff51-25bd-4c3e-a634-5980e8eb69a0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Enabled tasks FrameContext in call API",
```

Comment:
>tasks FrameContext [](http://example.com/codeflow?start=22&length=18)

I'm doing my best to channel my inner @erinha when reviewing this:

Could you change this to be:
"Enabled support `FrameContext.task` in `call` API
...so we can have the properly formatted code reference show up in the changelog?

Thanks!